.config/smtp.config.php

```diff
@@ -11,5 +11,9 @@
     'mail_smtppassword' => getenv('SMTP_PASSWORD') ?: '',
     'mail_from_address' => getenv('MAIL_FROM_ADDRESS'),
     'mail_domain' => getenv('MAIL_DOMAIN'),
+    'mail_smtpdebug' =>  getenv('SMTP_DEBUG') ?: false,
```

Comment:
Is there a use case for this in production?

Reply:
I don't know, I don't think, I think it was for completness, but no strong opinion, if you want we can remove.